analyzer/src/main/kotlin/curation/FilePackageCurationProvider.kt

```diff
@@ -79,7 +79,11 @@ class FilePackageCurationProvider(
             val allCurations = mutableListOf<Pair<PackageCuration, File>>()
 
             val curationFiles = paths.flatMap {
-                if (it.isDirectory) FileFormat.findFilesWithKnownExtensions(it) else listOf(it)
+                when {
```

Comment:
Have you considered the case where one explicitly specified a non-default file location? e.g. in such case should it be a program error if the file does not exist?

Reply:
Yeah, that's the problem highlighted by `FilePackageCurationProviderTest` now: In some cases (when the default location is used), we do *not* want an exception to be thrown. But if a non-default location that does not exist is specified, we (still?) want an exception to be thrown. With the unified code this becomes a bit nasty to implement. Let me think about it...